vllm/worker/worker.py

```diff
@@ -173,7 +173,9 @@ def _prepare_inputs(
             seq_id = seq_ids[0]
 
             seq_data = seq_group_metadata.seq_data[seq_id]
-            prompt_tokens = seq_data.get_token_ids()
+            
+            # prompt_tokens = seq_data.get_token_ids()
+            prompt_tokens = seq_data.get_token_ids_with_draft()
```

Comment:
I am wondering what if they user decide to not use SD? You force to call `get_token_ids_with_draft()` instead of `get_token_ids()`? Most of the case, when you try to add a important feature, you should make it compatible with original config, which means you should assume that we may want to disable this feature.